securedrop-workstation-config/debian/changelog-trusty

```diff
@@ -0,0 +1,5 @@
+securedrop-workstation-config (0.1.0) unstable; urgency=medium
+
+  * Initial release.
+
+ -- SecureDrop Team <securedrop@freedom.press>  Thu, 4 Oct 2018 15:33:10 -0400
```

Comment:
Remove `changelog-trusty` altogether. We could rename it to `changelog-stretch` as a historical reference, but since we're no longer building Stretch packages, it's just cruft in the repo at this point.